src/parca_agent.py

```diff
@@ -42,25 +88,26 @@ def remove(self):
         """Remove the Parca Agent snap, preserving config and data."""
         self._snap.ensure(snap.SnapState.Absent)
 
-    def configure(self, config, restart=True):
-        """Configure Parca Agent on the host system. Restart Parca Agent by default."""
-        self._snap.set({"remote-store-address": config.get("remote-store-address", "")})
-        self._snap.set({"remote-store-insecure": config.get("remote-store-insecure", "false")})
-        self._snap.set({"remote-store-bearer-token": config.get("remote-store-bearer-token", "")})
+    @property
+    def _arch(self) -> str:
```

Comment:
this could also be a class property, since it's static, or even a global, as it's not really a parca property but a system one